tests/src/test/scala/pathy/path.scala

```diff
@@ -27,28 +27,22 @@ class PathSpecs extends Specification with ScalaCheck {
   import Path._
   import posixCodec._
 
-  "two directories" in {
-    unsafePrintPath(dir("foo") </> file("bar")) must_== "./foo/bar"
-  }
-
-  "file with two parents" in {
-    unsafePrintPath(dir("foo") </> dir("bar") </> file("image.png")) must_== "./foo/bar/image.png"
-  }
-
-  "file without extension" in {
-    unsafePrintPath(file("image") <:> "png") must_== "./image.png"
-  }
-
-  "file with extension" in {
-    unsafePrintPath(file("image.jpg") <:> "png") must_== "./image.png"
-  }
-
-  "printPath - ./../" in {
-    unsafePrintPath(parentDir1(currentDir)) must_== "./../"
-  }
-
-  "print and parse again should produce same Path" ! prop { path: AbsFile[Sandboxed] =>
-    parseAbsFile(printPath(path)).get must_== path
+  "print and parse again should produce same Path" >> {
```

Comment:
@mossprescott It bugged me that the hard coding of `/` was only detected during code review as opposed to detected by some tests. I was able to expose the bug through tests with this. :smile: